assets/onboarding/welcome/index.jsx

```diff
@@ -14,13 +14,27 @@ export const Welcome = () => {
 
 	const { goTo } = useQueryStringRouter();
 
-	const { data, submit, isBusy } = useOnboardingApi( 'welcome' );
+	const { usageTracking, isFetching, isSubmitting } = useSelect(
+		( select ) => ( {
+			usageTracking: select( 'sensei-setup-wizard' ).getUsageTracking(),
+			isFetching: select(
+				'sensei-setup-wizard'
+			).isFetchingUsageTracking(),
+			isSubmitting: select(
+				'sensei-setup-wizard'
+			).isSubmittingUsageTracking(),
+		} ),
+		[]
+	);
+	const { submitUsageTracking } = useDispatch( 'sensei-setup-wizard' );
+
+	const isBusy = isFetching || isSubmitting;
```

Comment:
I separated the `isFetching` and `isSubmitting`, so we can change the loading feedback for the user as we desire. 

Maybe while we are fetching (including the other screens), we could add a loading in the Stepper (this is prepared to receive a loading). - When we work in the navigation persistency we could check this.